Formula/avimetaedit.rb

```diff
@@ -0,0 +1,26 @@
+class Avimetaedit < Formula
+  desc "Tool for embedding, validating, and exporting of AVI files metadata"
+  homepage "https://mediaarea.net/AVIMetaEdit"
+  url "https://mediaarea.net/download/binary/avimetaedit/1.0.1/AVIMetaEdit_CLI_1.0.1_GNU_FromSource.tar.bz2"
+  version "1.0.1"
+  sha256 "3aa4ca6aed51c71684f3fd1bd3b4defc2f9e50a8bd0bdf10e55e86c8ba24aafd"
+
+  def install
+    cd "Project/GNU/CLI" do
+      system "./configure", "--disable-debug",
+                            "--disable-dependency-tracking",
+                            "--disable-silent-rules",
+                            "--prefix=#{prefix}"
+      system "make", "install"
+    end
+  end
+
+  test do
+    avi = "UklGRuYAAABBVkkgTElTVMAAAABoZHJsYXZpaDgAAABAnAAAlgAAAAAAAAAQCQAAAQAAAAAAAAABAAAAAAAQAA" \
+          "IAAAACAAAAAAAAAAAAAAAAAAAAAAAAAExJU1R0AAAAc3RybHN0cmg4AAAAdmlkc0k0MjAAAAAAAAAAAAAAAAAB" \
+          "AAAAGQAAAAAAAAABAAAABgAAAP////8AAAAAAAAAAAIAAgBzdHJmKAAAACgAAAACAAAAAgAAAAEADABJNDIwBg" \
+          "AAAAAAAAAAAAAAAAAAAAAAAABMSVNUEgAAAG1vdmkwMGRjBgAAABAQEBCAgA==".unpack("m")[0]
+    File.open("test.avi", "wb") { |f| f.write avi }
```

Comment:
```ruby
(testpath/"test.avi").write avi
```
should work fine here, I believe. Just a bit more Homebrew-standard.

Reply:
Updated, thanks.